feeds/components.json

```diff
@@ -0,0 +1,46 @@
+[
+  {
+    "name": "Preview.ConversationalCore.GeneralIntent",
```

Comment:
A lot of this information seems duplicative of metadata that should naturally already be within a .nuspec or package.json file. Would it not make more sense to largely have this be pointers to some backing package from whichever source, and then normalize the package metadata into a standard representation for the purposes of rendering in Composer, etc.?

Reply:
Ultimately we can get a lot of this information from an npm search, which does in turn get the data from the packages.  We will do this in an upcoming version of Composer once components are regularly being published under out controlled namespace and a tagging strategy has been determined.

For example:
https://registry.npmjs.org/-/v1/search?text=botbuilder&keywords:botframework&size=100&from=0&quality=0.65&popularity=0.98&maintenance=0.5

Meanwhile, having the information here takes the burden off the consumer to dereference each package and extract the information, something that would be very time consuming to do at runtime.

Reply:
I don't think it can be *that* time consuming surely? Given we have identifiers for each package, I can't imagine that hydrating the metadata per package can be more than ~250-500ms, which if you then throw parallelization and caching into the mix would eliminate the need for repeated retrievals. That ought to be a very negligible cost, particularly if it means I only ultimately have to specify my metadata in one location.

Reply:
Spoke offline - intention is to for 1.4 release to pull entire feed.